src/cart/crt_hg.h

```diff
@@ -97,6 +97,8 @@ struct crt_hg_context {
 };
 
 /* crt_hg.c */
+int crt_hg_get_protocol_info(const char *info_string, struct na_protocol_info **na_protocol_info_p);
+void crt_hg_free_protocol_info(struct na_protocol_info *na_protocol_info);
```

Comment:
these are not exposed to the rest of the daos. we need to have external functions in src/include/cart/api.h, something like crt_get_protocol_info()  / crt_free_protocol_info()